### DISCUSSION TITLE
Dot notation relationship and soft deleted models

### DISCUSSION THREAD

**Discussion by RX-77IT** [filamentphp/filament#3904] [Help] [answered]
Hi!
I'm starting to use soft deletes with Filament and it all works wonderfully... except for a little problem that I don't know how to solve.

I have a Resource and this Resource has only one associated image.
```
public function image()
{
  return $this->belongsTo(Image::class);
}
```

When I soft delete the Resource I also soft delete the associated image, but then the ImageColumn::make('image.url') on the Resource's table stops showing the image. I think it's caused by Laravel query builder ignoring soft deleted images.
Is there a way to make the ImageColumn work again?

**Comment by danharrin** [comment, marked as answer]
https://laraveldaily.com/belongsto-and-withtrashed-linking-to-deleted-row

**Reply by RX-77IT** [reply]
:facepalm: Of course it was that simple...

And here I thought I needed something like this (it was the first result in google):
`->with(['trials.activity' => function ($q) {
    $q->withTrashed();
}, 'trials.activity.goalcategories.goals.scoretype']);`

Thank you very much!
